src/groups/bmq/bmqtst/package/bmqtst.mem

```diff
@@ -1,3 +1,4 @@
 bmqtst_blobtestutil
+bmqtst_loggingallocator
```

Comment:
`bmqtst` doesn't depend on `bmqma`, andI don't want introduce this dependency by placing `LoggingAllocator` there